### PR TITLE
Karma Quest: added onwer name in requestor column

### PR DESCRIPTION
**Pull request by sohailamjad12**

### CODE REVIEW

File: project/ws/app/src/lib/routes/home/components/request-list/request-list.component.ts [hunk 1]
```diff
@@ -76,6 +80,10 @@ export class RequestListComponent implements OnInit {
 
   ngOnInit() {
     this.configSvc = this.activeRoute.snapshot.data['configService']
+    this.fullProfile = _.get(this.activeRoute.snapshot, 'data.configService')
+    if (this.fullProfile.userProfile && this.fullProfile.userProfile.rootOrgId) {
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/request-list/request-list.component.ts]: Please do additional check

Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/home/components/request-list/request-list.component.ts]: Please do additional check

Comment (sohailamjad12) [reply project/ws/app/src/lib/routes/home/components/request-list/request-list.component.ts]: added

Comment (sohailamjad12) [reply project/ws/app/src/lib/routes/home/components/request-list/request-list.component.ts]: added checks

**Pull request merged**